src/main/webapp/projectContainer/ConsentGroups.js

```diff
@@ -64,11 +159,10 @@ export const ConsentGroups = hh(class ConsentGroups extends Component {
     this.closeRequestClarification();
   };
 
-  loadConsentGroups = (rThis) => {
-    $('.consent-group-panel-body').hide();
+  collapseBtnAnimationListener() {
     $('.consent-accordion-toggle').on('click', function () {
-      var icon = $(this).children().first();
-      var body = $(this).parent().parent().next();
+      let icon = $(this).children().first();
```

Comment:
I think this could be 
```suggestion
      const icon = $(this).children().first();
```